src/main/kotlin/com/zepben/evolve/services/network/translator/NetworkCimToProto.kt

```diff
@@ -387,6 +396,47 @@ fun toPb(cim: OperationalRestriction, pb: PBOperationalRestriction.Builder): PBO
 
 fun OperationalRestriction.toPb(): PBOperationalRestriction = toPb(this, PBOperationalRestriction.newBuilder()).build()
 
+/************ IEC61968 infIEC61968 ASSET INFO ************/
```

Comment:
Can you please relocate these sections to above `metering` to keep it sorted alphabetically. In this case can you also rename the section as described in the overall PR comment